package.json

```diff
@@ -70,7 +70,7 @@
     "d3-request": "^1.0.6",
     "d3-scale": "^1.0.6",
     "deck.gl": "4.1.6",
-    "dom-to-image": "github:heshan0131/dom-to-image",
+    "dom-to-image": "https://git@github.com/heshan0131/dom-to-image.git",
```

Comment:
Looks fine to me. Do we have a PR to merge your changes to dom-to-image? Perhaps we can track that as an issue.

These Git dependencies can slow down installs on Codepen/code playground websites etc.